owasp-top10-2017-apps/a1/copy-n-paste/app/util/db.go

```diff
@@ -46,8 +46,7 @@ func AuthenticateUser(user string, pass string) (bool, error) {
 	}
 	defer dbConn.Close()
 
-	query := fmt.Sprint("select * from Users where username = '" + user + "'")
-	rows, err := dbConn.Query(query)
+	rows, err := dbConn.Query("select * from Users where username =?", username)
```

Comment:
Hey, @Calegp ! Your mitigation solution sounds good to me! You just need to fix this typo: `username` -> `user` 😃